docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -98,15 +98,41 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are Scheduled
 
-Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler.  However, Pods
+### Scheduled by DaemonSet controller (default)
+
+Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler. However, Pods
 created by the DaemonSet controller have the machine already selected (`.spec.nodeName` is specified
-when the Pod is created, so it is ignored by the scheduler).  Therefore:
+when the Pod is created, so it is ignored by the scheduler). Therefore:
 
  - The [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node is not respected
    by the DaemonSet controller.
```

Comment:
When switching to default scheduler, does default scheduler check DS pods and make sure DS pods can still be scheduled on `unschedulable` nodes? If so, mention it in the below `Scheduled by default scheduler` section too.

Reply:
Added this as a note.